src/app/core/models/user-budgets/user-budgets.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Price } from 'ish-core/models/price/price.model';
+
+export interface UserBudgets {
```

Comment:
Please move this model to the organization-management model folder. We shouldn't be needing it in the core PWA.